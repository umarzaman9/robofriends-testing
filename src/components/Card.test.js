import React from 'react';
import {shallow} from 'enzyme';
import Card from './Card';

 
it('Card Component', () => {
expect(shallow(<Card />)).toMatchSnapshot();	
})