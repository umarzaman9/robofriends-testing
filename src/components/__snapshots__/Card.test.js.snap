// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component 1`] = `ShallowWrapper {}`;
